lib/auth/methods.go

```diff
@@ -161,7 +168,8 @@ func (a *Server) emitAuthAuditEvent(ctx context.Context, props authAuditProps) e
 			Success: true,
 		},
 		UserMetadata: apievents.UserMetadata{
-			User: props.username,
+			User:       props.username,
+			UserOrigin: props.userOrigin,
```

Comment:
The integer value will be reflected in the UI audit event. Not sold out on the alternative to use string field for audit event and the use int for the usage event. Open for suggestions.

I noticed we emit integer value for `user_kind` field.